Dashboard/app/js/lib/views/views.js

```diff
@@ -555,11 +555,10 @@ FLOW.DateField2 = Ember.TextField.extend({
     this._super();
 
     this.$().datepicker({
-      dateFormat: 'yy/mm/dd',
+      dateFormat: 'yy-mm-dd',
```

Comment:
You are right. I switched back to master and the problem persists. I'm not sure where the root issue resides